Source/MediaInfo/Text/File_Pdf.cpp

```diff
@@ -260,7 +260,10 @@ void File_Pdf::xref()
         {
             if (*Buffer_Temp=='n')
             {
-                int32u Offset=(int32u)atoi((const char*)(Buffer_Temp-17));
+                char atoi_buf[30];

```

Comment:
30 --> 18 please (else there is a risk to go beyond the buffer size during copy), and maximal 17 bytes expected for the number.

BTW, as I was also working on it, on my side I added too:
            if (Buffer_Temp+20>=Buffer+Buffer_Size)
                break; //Problem
before  if (*Buffer_Temp=='n')
please add theses lines so only 1 PR for this issue.